content/posts/jellyfin-10-5-0.md

```diff
@@ -0,0 +1,170 @@
+
+## Server
+
+This cycle, our server team ported the code over to .NET Core 3.1. This move makes new features available to our developers, including support for ARM64 for Linux, compatibility with TLS v1.3 and better garbage collection on Docker, allowing for better memory usage when running in a container.
+
+A lot of code cleanup and refactoring was done, fixing a few issues and bringing some speed improvements. Among these, the following table, while a little technical, shows the impressive speed improvement between the previous implementation of our hex decoder and the one shipped in Jellyfin 10.5.0.
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+Another example of the speed improvements to the server this cycle is the following comparison between our previous alphanumeric comparator and the one shipped in Jellyfin 10.5.0, showing a reduction of the execution time of more than 50%!
+
+<table style="width:100%">
+<thead>
+    <th>Method</th>
+    <th>Mean</th>
+    <th>Error</th>
+    <th>StdDev</th>
+    <th>Gen 0</th>
+    <th>Allocated</th>
+</thead>
+<tbody>
+<tr>
+    <td><b>Jellyfin 10.5.0</b></td>
+    <td style="text-align:right">10.81 μs</td>
+    <td style="text-align:right">0.079 μs</td>
+    <td style="text-align:right">0.074 μs</td>
+    <td style="text-align:right">0.1068</td>
+    <td style="text-align:right">336 B</td>
+</tr>
+<tr>
+    <td><b>Jellyfin 10.4.3</b></td>
+    <td style="text-align:right">23.57 μs</td>
+    <td style="text-align:right">0.055 μs</td>
+    <td style="text-align:right">0.051 μs</td>
+    <td style="text-align:right">11.2305</td>
+    <td style="text-align:right">35272 B</td>
+</tr>
+</tbody>
+</table>
+
+**Support AMD AMF hardware encoding** is now available on Windows and Linux, as well as support for DVDs as folders, a provider for season images for TheMovieDB and various fixes to media scanning and the way base URLs and DLNA are handled. **Raspberry Pi hardware acceleration** for older models is also now supported. Sorry Raspberry Pi 4 owners, this one needs a bit more work to be supported properly.
```

Comment:
> Sorry Raspberry Pi 4 owners, this one needs a bit more work to be supported properly.

So H264 HWA Encoding (the really important part) is enabled on Linuxserver Docker on Raspbian and the normal repository method on Raspbian. I have not gotten it to work on any other OS nor our docker image.

Reply:
So I could change it like this:

```suggestion
**Support AMD AMF hardware encoding** is now available on Windows and Linux, as well as support for DVDs as folders, a provider for season images for TheMovieDB and various fixes to media scanning and the way base URLs and DLNA are handled. **Raspberry Pi hardware-accelerated encoding** for H264 now supported on Raspbian, both using Docker and the repository package on Raspbian.
```

That would make the few restrictions clear.

Reply:
specifically for RPi4. I think Rpi3 can do decoding but @ferferga would know more for the Rpi3. and it's only available on LinuxServer, not our docker image.

Reply:
```suggestion
**Support for AMD AMF hardware encoding** is now available on Windows and Linux, as well as support for DVDs as folders, a provider for season images for TheMovieDB and various fixes to media scanning and the way base URLs and DLNA are handled. **Raspberry Pi hardware acceleration** for older models is also now supported. For Raspeberry Pi 4, hardware-accelerated encoding for H264 is now supported on Raspbian, both using the LinuxServer.io Docker and the repository package
```

This seems more correct, then.

Reply:
looks good :)